src/probability_theory/stopping.lean

```diff
@@ -0,0 +1,294 @@
+/-
+Copyright (c) 2021 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying
+-/
+import measure_theory.function.conditional_expectation
+import probability_theory.notation
```

Comment:
Those imports feel weird: I don't see any use of the conditional expectation in this file. Could you change them to what is actually used?